runtime/codegen/listeners.go

```diff
@@ -59,7 +59,7 @@ type ComponentListeners struct {
 // MakeListenersString() in data.
 func ExtractListeners(data []byte) []ComponentListeners {
 	var results []ComponentListeners
-	re := regexp.MustCompile(`⟦([0-9a-fA-F]+):wEaVeRlIsTeNeRs:([a-zA-Z0-9\-.~_/]*?)→([a-zA-Z0-9\-.~_/,]*?)⟧`)
+	re := regexp.MustCompile(`⟦([0-9a-fA-F]+):wEaVeRlIsTeNeRs:([a-zA-Z0-9\-.~_/]*?)→([\p{L}\d_,]+)⟧`)
```

Comment:
Should we use `\p{Nd}` instead of `\d` to match [unicode digits ](https://go.dev/ref/spec#unicode_digit) instead of just 0-9?